migration/migrating-openshift-3-to-4.adoc

```diff
@@ -0,0 +1,92 @@
+[id="migrating-openshift-3-to-4"]
+= Migrating {product-title} version 3 to 4
+include::modules/common-attributes.adoc[]
+:context: migrating-openshift-3-to-4
+
+toc::[]
+
+You can migrate application workloads from {product-title} 3.7 to a later version with the Cluster Application Migration (CAM) tool. The CAM enables you to control the migration and to minimize application downtime. The CAM provides a web console and an API, based on Kubernetes custom resources, for migrating stateful application workloads, at the granularity of a namespace, from a source to a target cluster.
```

Comment:
> This first sentence makes it sound like you can only migrate from version 3.7 exactly. Suggest rewording
> 
> `You can migrate application workloads from {product-title} 3.7 and later to {product-title} 4.x with the...`

Actually, 3,7+ can be located to any later version, not just 4.x. I'll add 'and later' to 3.7. But that is closely followed by "to a later version". Will have to think about that.

Reply:
So my thoughts are that, even if you can use the tool for 3.x -> 3.x, these docs are living in the OCP 4.2 doc set. So we should assume that's the version users are on (or going to).

I don't think we'd want to get into users using these docs when working with 3.x -> 3.x migrations.

Reply:
True. It's a problem that has never really been resolved. The doc is about migrating to 4.2 but they want users to know that it's possible to migrate to 3.x. I'll see whether I can put that into a note somewhere, so that the 3 > 4.2 path is clearer.